blockbase/style-editor.css

```diff
@@ -0,0 +1,85 @@
+/*--------------------------------------------------------------
+# Base Styles (Editor)
+#
+# These provide a baseline for editor styles. 
+# Please override these styles in editor-custom.css.
+--------------------------------------------------------------*/
+
+/*--------------------------------------------------------------
+# Base Styles
+--------------------------------------------------------------*/
+
+.editor-post-title__block .editor-post-title__input {
+	font-family: var(--wp--typography--font-family-heading);
+	color: var(--wp--color--text);
+}
+
+body,
+.editor-styles-wrapper {
+	background: var(--wp--color--background);
+	color: var(--wp--color--text);
+}
+
+body > *,
+.editor-styles-wrapper > *,
+.editor-styles-wrapper p {
+	font-family: var(--wp--typography--font-family);
+	font-size: var(--wp--typography--font-size);
```

Comment:
Right now this is being overridden in the editor by two higher-precedence style rules injected by Gutenberg: 

<img width="311" alt="Screen Shot 2020-04-10 at 4 26 18 PM" src="https://user-images.githubusercontent.com/5375500/79020996-66284b80-7b48-11ea-9b08-2b5d9d2a24af.png">